flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -847,6 +854,26 @@ public TableResultInternal executeInternal(List<ModifyOperation> operations) {
                 executeInternal(ctasOperation.getCreateTableOperation());
                 mapOperations.add(ctasOperation.toSinkModifyOperation(catalogManager));
             } else {
+                boolean isRowLevelModification = isRowLevelModification(modify);
+                if (isRowLevelModification) {
+                    String modifyType =
+                            ((SinkModifyOperation) modify).isDelete() ? "DELETE" : "UPDATE";
+                    if (operations.size() > 1) {
+                        throw new TableException(
+                                String.format(
+                                        "Unsupported SQL query! Only accept a single SQL statement of type %s.",
+                                        modifyType));
+                    }
+                    if (isStreamingMode) {
+                        throw new TableException(
```

Comment:
Hi, @luoyuxia is there any document about this not supported statement(how to migrate them from previous versions)? and will we support this in the near future? thanks.

Reply:
Thanks reach me out. What do you mean by saying `how to migrate them from previous versions`? Currently,it's by design that delete & update is not supported in stream mode  and the previous versioneven don't support delete & update statement. So, what is for migration?

Reply:
@luoyuxia thanks for the reply, and sorry for the previous message. after another check, found that the problem is not caused here, will start discuss in the third project.